Auto-fav was renamed to auto-show (#3890)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/create-an-org-wide-team.md

 Reduce @mentions to keep them from overloading the entire organization. Go to the team and click **˙˙˙ More options** > **Manage Team**. On the **Settings** tab, click <strong>@mentions</strong> > turn off **Show members the option to @team or @[team name]**. 

### Automatically show important channels

> Show important channels to ensure everyone in your organization engages in specific conversations. To learn more, see [Auto-favorite channels for the whole team](https://support.office.com/article/auto-favorite-channels-for-the-whole-team-a948272c-5aa5-429c-863c-4e1e1cd6b0f6). Note: This feature was recently renamed from Automatically favorite to Automatically show.

Reviewer: ```suggestion
Show important channels to ensure everyone in your organization engages in specific conversations.
> [!NOTE]
> This feature was previously named Automatically favorite.
```
